src/components/ChatApp/SearchField.react.js

```diff
@@ -140,14 +140,24 @@ class ExpandingSearchField extends Component {
           <span className="counter">
             {this.props.searchIndex}/{this.props.searchCount}
           </span>
-          <IconButton
-            className="displayNone"
-            iconStyle={baseStyles.smallIcon}
-            style={baseStyles.icon}
-            onClick={this.onClickPrev}
-          >
-            <UpIcon />
-          </IconButton>
+          {this.props.searchText === '' ? (
```

Comment:
Strings need to be in double quotes. Probably the reason why Codacy tests are failing.
```suggestion
          {this.props.searchText === "" ? (
```